### PR TITLE
saapby

### PR DESCRIPTION
**Pull request by saapby**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -15,18 +15,27 @@
         <allure.version>1.4.19</allure.version>
         <aspectj.version>1.8.5</aspectj.version>
     </properties>
-    
+
     <dependencies>
+
```
Comment (Kreisfahrer) [inline pom.xml:20]: you should not change pom, at least without reason

File: src/main/java/core/BmpTestBase.java [hunk 2]
```diff
@@ -41,4 +42,8 @@ public void cleanup() {
         saveEnvironment();
     }
 
+
+    public ProxyServer getServer() {
```
Comment (Kreisfahrer) [inline src/main/java/core/BmpTestBase.java:46]: Let server field be protected instead

File: src/main/java/core/TestBase.java [hunk 3]
```diff
@@ -21,19 +18,6 @@
 
     @BeforeTest
     public void configure() throws Exception {
-        String cloud = System.getProperty("browserStack", "");
-        if (cloud.length() > 0) {
```
Comment (Kreisfahrer) [inline src/main/java/core/TestBase.java:25]: get the latest from master before creating pool request

File: src/test/java/BrokenImageBMPTest.java [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+    @BeforeMethod
+    public void setup() {
+        bmp = getServer();
+        open("broken_images");
+    }
+
+    @Test
+    public void brokenImageTest() {
+        List<String> links = getUrls($$(".example>img"), "src");
```
Comment (Kreisfahrer) [inline src/test/java/BrokenImageBMPTest.java:27]: That is totally not your case. Have a look at net panel in firebug when the the pages is loaded - you will see the responses that failed -> so check them in hard

File: pom.xml (hunk 1 above)
Comment (saapby) [inline pom.xml:20]: Ok. I will consider it on the future.

**Pull request closed without merging**